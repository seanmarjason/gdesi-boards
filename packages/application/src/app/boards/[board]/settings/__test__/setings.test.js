import { render } from '@testing-library/react'
import BoardSettings from '../settings';

jest.mock("next/navigation");

const board = {
    id:1,
    name:"Operations",
    manager:2,
    users:[1,3,2],
    columns:[
        {id:1,title:"To Do",cards:[]},
        {id:2,title:"Doing",cards:[]},
        {id:3,title:"Done",cards:[]}
    ]
}

global.fetch = jest.fn(() =>
    Promise.resolve({
      json: () => Promise.resolve(board),
    }),
  );

it('BoardSettings renders correctly', async () => {
  const { container } = render(<BoardSettings boardId={1}/>)
  expect(container).toMatchSnapshot()
})


