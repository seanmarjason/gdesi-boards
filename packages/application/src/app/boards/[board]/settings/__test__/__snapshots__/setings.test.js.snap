// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BoardSettings renders correctly 1`] = `<div />`;
